int64 alalaser0
int64 alalaser1
int64 sever0
int64 sever1

